capstone-api/models/User.js

```diff
@@ -308,6 +321,64 @@ class User {
     query.equalTo("username", username).descending("createdAt");
     return await query.find();
   }
+
+  static async getMembers(groupName) {
+    const query = new Parse.Query("UserGroup");
+    query.equalTo("groupName", groupName);
+    return (await query.find({})).map((element) => {
+      return {
+        username: element.get("username"),
+        isAdmin: element.get("isAdmin"),
+      };
+    });
+  }
+
+  static async setGroupGenres(groupName, genres) {
+    const query = new Parse.Query("Group");
+    query.equalTo("name", groupName);
+    let result = await query.first({});
```

Comment:
similarly, avoid result as a variable name

Reply:
in later PR